guides/installing-ionic-framework.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Installing Ionic Framework
+author: Sergiu Butnarasu
```

Comment:
Can you replace this with your koding username? We'll automatically link it to your koding profile that way.